openeogeotrellis/utils.py

```diff
@@ -572,3 +574,89 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 @functools.lru_cache
 def is_package_available(name: str) -> bool:
     return any(m.name == name for m in pkgutil.iter_modules())
+
+
+def reproject_cellsize(
+        spatial_extent: dict,
+        native_resolution: dict,  # cell_width, cell_height, crs
```

Comment:
Without more context, it is not clear what "native" means here. This function is just about converting cell size, but there are no assumptions about what is native. I would just call this  `cell_size`.

also note that the current API is a bit asymmetric: the input cell size should be a dictionary, but the output cell size is a tuple

Reply:
I split the `native_resolution` into `input_resolution` and `input_crs`